<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>800</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="50,50">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="10,40,50">
          <property name="topMargin">
           <number>5</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinAndMaxSize</enum>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="50,50">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_10" stretch="10,90">
                  <property name="sizeConstraint">
                   <enum>QLayout::SetFixedSize</enum>
                  </property>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_12">
                    <property name="sizeConstraint">
                     <enum>QLayout::SetFixedSize</enum>
                    </property>
                    <property name="leftMargin">
                     <number>5</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="maximumSize">
                       <size>
                        <width>300</width>
                        <height>50</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>14</pointsize>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color:lightgreen</string>
                      </property>
                      <property name="text">
                       <string>Départ:</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_11">
                    <property name="sizeConstraint">
                     <enum>QLayout::SetFixedSize</enum>
                    </property>
                    <property name="leftMargin">
                     <number>5</number>
                    </property>
                    <item>
                     <widget class="QLineEdit" name="lineEdit">
                      <property name="enabled">
                       <bool>true</bool>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>14</pointsize>
                       </font>
                      </property>
                      <property name="mouseTracking">
                       <bool>false</bool>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">border-radius:20px</string>
                      </property>
                      <property name="text">
                       <string>Le Bourget</string>
                      </property>
                      <property name="frame">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_13" stretch="10,90">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_14">
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="maximumSize">
                       <size>
                        <width>300</width>
                        <height>50</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>14</pointsize>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color:red</string>
                      </property>
                      <property name="text">
                       <string>Arrivée:</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_15">
                    <item>
                     <widget class="QRadioButton" name="radioButton_Bourget">
                      <property name="font">
                       <font>
                        <pointsize>12</pointsize>
                        <kerning>true</kerning>
                       </font>
                      </property>
                      <property name="text">
                       <string>Le Bourget</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="radioButton_NY">
                      <property name="font">
                       <font>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true"/>
                      </property>
                      <property name="text">
                       <string>New-York</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="radioButton_Vladivostock">
                      <property name="font">
                       <font>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true"/>
                      </property>
                      <property name="text">
                       <string>Vladivostock</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_19" stretch="10,90">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_22">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Période de départ souhaitée:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_18">
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>623</width>
                  <height>228</height>
                 </size>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <property name="sizeConstraint">
                    <enum>QLayout::SetDefaultConstraint</enum>
                   </property>
                   <property name="leftMargin">
                    <number>10</number>
                   </property>
                   <property name="rightMargin">
                    <number>10</number>
                   </property>
                   <item>
                    <widget class="QRadioButton" name="radioButton_Printemps">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>200</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="autoFillBackground">
                      <bool>false</bool>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">spacing: 10px;


</string>
                     </property>
                     <property name="text">
                      <string>Printemps</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="radioButton_Ete">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>200</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="cursor">
                      <cursorShape>ArrowCursor</cursorShape>
                     </property>
                     <property name="autoFillBackground">
                      <bool>false</bool>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">spacing: 10px;</string>
                     </property>
                     <property name="text">
                      <string>Eté</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="radioButton_Automne">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>200</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">spacing: 10px;</string>
                     </property>
                     <property name="text">
                      <string>Automne</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="radioButton_Hiver">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>200</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">spacing: 10px;</string>
                     </property>
                     <property name="text">
                      <string>Hiver</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_saison">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>200</width>
                     <height>200</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap resource="images.qrc">:/images/images/printemps.png</pixmap>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7" stretch="10,90">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_17">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Poids de la charge utile:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_16">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_20">
                <item>
                 <widget class="QLabel" name="label_dirigeable">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>650</width>
                    <height>150</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap resource="images.qrc">:/images/images/blimp-moyen.png</pixmap>
                  </property>
                  <property name="scaledContents">
                   <bool>true</bool>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLCDNumber" name="lcdNumber_charge">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>8</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="horizontalSlider">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>500</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="minimum">
                   <number>0</number>
                  </property>
                  <property name="maximum">
                   <number>200</number>
                  </property>
                  <property name="singleStep">
                   <number>1</number>
                  </property>
                  <property name="pageStep">
                   <number>1</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="tickPosition">
                   <enum>QSlider::TicksAbove</enum>
                  </property>
                  <property name="tickInterval">
                   <number>20</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="60,10,10">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="topMargin">
           <number>5</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9" stretch="90">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="50,50">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_21">
              <item>
               <widget class="QPushButton" name="pushButton_DimAuto">
                <property name="maximumSize">
                 <size>
                  <width>1280</width>
                  <height>800</height>
                 </size>
                </property>
                <property name="text">
                 <string>Dimensionnement  
 Automatique</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_20">
              <item>
               <widget class="QPushButton" name="pushButton_DimManu">
                <property name="maximumSize">
                 <size>
                  <width>1280</width>
                  <height>800</height>
                 </size>
                </property>
                <property name="text">
                 <string>Dimensionnement 
 Manuel</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QPushButton" name="pushButton_Retour">
                <property name="maximumSize">
                 <size>
                  <width>1280</width>
                  <height>800</height>
                 </size>
                </property>
                <property name="text">
                 <string>Retour</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
